lib/PuppeteerSharp/Input/Key.cs

```diff
@@ -0,0 +1,112 @@
+﻿namespace PuppeteerSharp.Input
+{
+    public class Key
```

Comment:
I don't find the usage of this class @Meir017.

Reply:
it's a utility class - https://github.com/kblok/puppeteer-sharp/pull/180#discussion_r185612378